### PR TITLE
arista_mos-support

### PR DESCRIPTION
**Pull request by hanserasmus**
Add initial support for correct arista-mos support.

I think my tests files are named incorrectly. If anyone could please indicate what the correct line should be for running the collect and save, I would gladly redo it and resubmit.

Most of the data found on these devices are from standard SNMPv2 MIB, they have no special MIBS published as far as  I can tell, and they only reference standard ones in their documentation. The hardware version for example is not published in these standard mibs. I can submit a full walk if needed be.

image

image

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/arista_mos.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'CPU Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+discovery:
+    -
+        sysObjectID:
+            - .1.3.6.1.4.1.43191.1.6.7
+poller_modules:
+    wireless: false
```
Comment (PipoCanaja) [inline includes/definitions/arista_mos.yaml:15]: Hi @hanserasmus 
Could you please avoid disabling modules, unless they fail on your device.

**Review by PipoCanaja — CHANGES_REQUESTED** [review]

**Comment by PipoCanaja** [comment]
In term of naming I would : 
- name the os arista-mos
- name the yaml files arista-mos.yaml
- name the files tests/data/arista-mos_metamux48.json and tests/snmprec/arista-mos_metamux48.snmprec

**Comment by hanserasmus** [comment]
Hi @PipoCanaja I can do all of those, definitely. I will rerun the tests with the new filenames, and I will rename the files as requested. I will also remove those modules. I saw them as definitions in arista_eos so thought it would be wise to use the same settings.

Will change and resubmit shortly.

**Review by PipoCanaja — APPROVED** [review]
LGTM !

**Comment by PipoCanaja** [comment]
Thanx @hanserasmus

**Pull request merged**